doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -51,6 +51,7 @@ Bootloaders and DFU
 * Added documentation for :ref:`qspi_xip_split_image` functionality.
 * Added a section in the sysbuild-related migration guide about the migration of :ref:`child_parent_to_sysbuild_migration_qspi_xip` from child/parent image to sysbuild.
 * Removed secure bootloader Kconfig ``CONFIG_SECURE_BOOT_DEBUG`` and replaced with usage of logging subsystem.
+* Siggned applcation image intel-hex file built along with MCUBoot in diret-XPI with revert mode gets confirmation flag automatically. No need for dedicated builds for flashing the device and ubdate the device anymore.
```

Comment:
```suggestion
* Added: 
  
  * Documentation for :ref:`qspi_xip_split_image` functionality.
  * A section in the sysbuild-related migration guide about the migration of :ref:`child_parent_to_sysbuild_migration_qspi_xip` from child/parent image to sysbuild.
 
* Updated the signed application image in Intel HEX format to include MCUBoot in direct-xip mode with the revert support, ensuring it automatically receives a confirmation flag.
  This adjustment eliminates the need for separate builds for device flashing and updates.

* Removed secure bootloader Kconfig ``CONFIG_SECURE_BOOT_DEBUG`` and replaced with usage of logging subsystem.
```

Edited the structure of the entire section. Please double-check the meaning of the "updated" item

Reply:
the middle sentence meaning differs, let's rephrase:
```
Updated the procedure of signing of the application image which is build for being booted by MCUboot in directXIP mode with revert support, so the application image Intel-Hex fil automatically receives a confirmation flag.
```

Reply:
Noted, thanks! Would it be okay to rephrase it like that (?)

Updated the procedure for signing the application image built for booting by MCUboot in directXIP mode with revert support. 
Now, the Intel-Hex file of the application image automatically receives a confirmation flag.

Reply:
Taht's works, thanks!